bins/packages/storcli/storcli.sh

```diff
@@ -1,17 +1,18 @@
+STORCLI_VERSION="1.21.06"
+
 dependencies_storcli() {
     apt-get update
-    apt-get install wget
 
     echo "[+] updating soruces list"
     wget -O - http://archive.thomas-krenn.com/tk-archive.gpg.pub | apt-key add -
```

Comment:
Did you tested your script ? Because it doesn't works on a fresh Ubuntu 18.04 docker. Which mean it won't work neither on the github workflow. The `apt-key` command fails, you need to install `gnupg` (`apt-get install gnupg`) first.